policy/modules/system/systemd.te

```diff
@@ -581,11 +581,13 @@ init_read_script_files(systemd_generator_t)
 init_getattr_all_unit_files(systemd_generator_t)
 init_getattr_all_script_files(systemd_generator_t)
 
+# vmware_vsock
+kernel_request_load_module(systemd_generator_t)
```

Comment:
Nit: please move to the end of the `kernel_*` lines (after `kernel_dontaudit_search_unlabeled`)

Reply:
moved